using-mobile-extensions/adobe-journey-optimizer/api-reference.md

```diff
@@ -1,5 +1,41 @@
 # Adobe Journey Optimizer API reference
 
+## addPushTrackingDetails (Android only) <a id="addpushtrackingdetails"></a>
+
+The `addPushTrackingDetails` API is used to update a pending intent with important information, such as messageId and Customer Journey information.
+
+**Note:** Calling this API is mandatory, so the pending intent can be used while tracking push notification interactions.
+
+{% tabs %}
+
+{% tab title="Android" %}
+
+#### Java
+
+**Syntax**
+
+```java
+public static boolean addPushTrackingDetails(final Intent intent, final String messageId, final Map<String, String> data)
+```
+
+| Variable    | Type                  | Description                                                  |
+| :---------- | :-------------------- | :----------------------------------------------------------- |
+| `intent`    | `Intent`              | The pending intent that needs to be updated so it can be used when the user interacts with the notification. |
+| `messageId` | `String`              | The message ID for the push notification.                    |
+| `data`      | `Map<String, String>` | The data of the remoteMessage.                               |
```

Comment:
I think we normally do these as a list, rather than a table, so for consistency across the documentation, this should also be a list instead.

(See, https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/identity/identity-api-reference as an example).

Reply:
another consequence of the migration. i'll get this fixed as well.